### DISCUSSION TITLE
Problem preservating the Amiga version of "Ivanhoe" by Ocean

### DISCUSSION THREAD

**Discussion by realmml** [keirf/disk-utilities#139] [Q&A] [answered]
Other thread another game, this is really strange, because the original floppy works great in the Amiga, and the output by disk-analyse doesn't show anything strange as you can see:
```disk-analyse.exe -f probe_amiga convert\ivanhoe\track ivanhoe.ipf
T0.0-79.1: AmigaDOS
T80.0-83.1: Unformatted
```

I think the dump must be bad, because the IPF boot load a little and then remains in a black screen forever. The dump can be downloaded here:

[Ivanhoe Dump](https://www.dropbox.com/s/prtdvk39tfdyx1o/ivanhoe.zip)

**Comment by kkrellwitz** [comment]
I took a quick look at this and the dump appears to be good, but it looks like there is a protection check against track 1.  Using probe_amiga it gets decoded as a standard amigados track.  I will dig into this further, but it may take some time as I do not have a lot of free time at the moment.

If you create a ctraw image using the Kryoflux software, the game loads fine in WinUAE.

**Comment by keirf** [comment, marked as answer]
Must be a gap protection of some sort. Identifying as AmigaDOS but then not working is the classic sign of this. Intentionally so!

Since it's an Ocean title there's a fighting chance it is `rnc_gap`. Give that a go.

**Reply by kkrellwitz** [reply]
Just gave it a go and It is indeed rnc_gp. 

I will update the format file later with the following entry:

"Ivanhoe"
    0 amigados
    1 rnc_gap
    2-159 amigados
    * ignore

**Reply by keirf** [reply]
Done, thanks! There is a shorter valid form for this common protection:
```
"Ivanhoe" = rnc_gap
```

**Comment by realmml** [comment]
Thanks! :)
